astropy/utils/introspection.py

```diff
@@ -111,6 +121,19 @@ def minversion(module, version, inclusive=True, version_path='__version__'):
     True
     """
 
+    if isinstance(module, types.ModuleType):
+        module_name = module.__name__
+    elif isinstance(module, six.string_types):
+        module_name = module
+        try:
+            module = resolve_name(module_name)
+        except ImportError:
+            return False
```

Comment:
It's just so that I can define these constants regardless of whether Numpy is installed or not.  I see what you're saying about `NUMPY_LT_1_9` being `True` being confusing though.  I think that's more a problem with the naming of that constant.  In any case that only becomes an issue in cases where Numpy is actually not working, and that would be caught in the actual import statements well before the values of those constants matter.

It might be less of an issue if they were named more in accordance with the semantics of `minversion`.  So instead of `NUMPY_LT_1_9` have `NUMPY_AT_LEAST_1_9` and so on.  I just wanted to keep the number of changes at a minimum.